### PR TITLE
remove tests that query by mcountry,channels,locales m2m fields

### PR DESCRIPTION
**Pull request by peterbe**
Part of #1524 

Once this lands we can remove the serialization of Channel, Locale and Countries from the `recipes/api/v*/serializers.py`.

### CODE REVIEW

File: normandy/recipes/tests/test_models.py [hunk 1]
```diff
@@ -447,79 +405,11 @@ def test_recipe_revise_partial(self):
...
-        recipe.revise(name="changed", action=a2, channels=[c])
+        recipe.revise(name="changed", action=a2)
         assert recipe.action == a2
         assert recipe.name == "changed"
         assert recipe.arguments == {"message": "something"}
-        assert recipe.filter_expression == (
-            "(normandy.channel in ['beta']) && " "(something !== undefined)"
-        )
-
-    def test_recipe_doesnt_revise_when_clean(self):
```
Comment (mythmon) [inline normandy/recipes/tests/test_models.py]: I think this test is still valid, just without the channels part.

Comment (peterbe) [reply normandy/recipes/tests/test_models.py]: Restored, fixed and rebased (new commit message)

Comment (mythmon) [reply normandy/recipes/tests/test_models.py]: Maybe there was a different typo that I didn't see, but I think "mcountry" in the commit message isn't right.

**Comment by peterbe** [comment]
Now with a separate commit to remove the serialization.

File: normandy/recipes/tests/test_models.py (hunk 1 above)
Comment (peterbe) [reply normandy/recipes/tests/test_models.py]: There. Fixed now.

**Comment by mythmon** [comment]
:red_circle: :red_circle: :red_circle: 

Thanks for taking this all out, I'm excited about the rest of it too.

**Comment by peterbe** [comment]
bors retry

**Pull request merged**